README.md

```diff
@@ -32,32 +32,27 @@ bundle
 bundle exec rake webpacker:compile
 ```
 
-### 3. Configure DfE Sign-In
+### 3. Setup Basic Auth Login
 
-Create the following file and ask the team for the secret.
+For local development, we rely on Basic Auth instead of DFE Sign-In, which can
+be enabled if required. Create the file `config/settings/development.local.yml`
+with the contents:
 
+```yaml
+authorised_users:
+  0:
+    first_name: [your first name, this will be updated in the db]
+    last_name: [your last name, this will be updated in the db]
+    email: [the email address to login with]
+    password: [the password you wish to use]
 ```
-# config/settings/development.local.yml
-dfe_signin:
-    secret: dfe_sign_in_test_server_client_secret_here
-```
-
-*If you don't have a sign-in account on test you can bypass sign-in by
-configuring basic auth instead.*
 
+The email address has to exist in the users table of teacher-training-api, which
+means that you'll need to have an account on the production system (or add
+yourself by hand), but the password can be any non-secure local password you
```

Comment:
Do you think it would be useful to add an example of adding a user by hand since there is a couple of things to include.

```
User.create(admin: true, email: "john.smith@digital.education.gov.uk", 
first_name: "John", last_name: "Smith", welcome_email_date_utc: Time.now,
accept_terms_date_utc: Time.now, invite_date_utc: Time.now, state: "transitioned")
```

Reply:
Sure can. 👍

Reply:
@AgaDufrat Done.